tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -601,3 +602,11 @@ def test_call_max_var_criterion_with_dataset_by_default(mocker, mode):
     compress_weights(model, mode=mode, ratio=0.8, group_size=-1, dataset=dataset)
 
     scores_spy.assert_called()
+
+
+@pytest.mark.parametrize("mode", INT4_NF4_MODES)
+def test_call_max_var_criterion_with_dataset_by_default_awq(mode):
+    model = AWQMatmulModel().ov_model
+    dataset = Dataset([np.ones([8, 8])])
+
+    compress_weights(model, mode=mode, ratio=1.0, group_size=2, dataset=dataset, awq=True)
```

Comment:
Is it possible to check that the model's outputs are the same before and after scaling weights?